docs/01-app/03-api-reference/05-config/02-typescript.mdx

```diff
@@ -133,9 +135,15 @@ Next.js will generate a link definition in `.next/types` that contains informati
 
 > **Good to know**: If you set up your project without `create-next-app`, ensure the generated Next.js types are included by adding `.next/types/**/*.ts` to the `include` array in your `tsconfig.json`:
 
-```json filename="tsconfig.json"
+```json filename="tsconfig.json" highlight={4}
 {
-  "include": ["next-env.d.ts", ".next/types/**/*.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    ".next/types/**/*.ts",
+    "**/*.ts",
+    "**/*.tsx"
+    // other paths in your configuration
```

Comment:
Mmm I had added this to be able to highlight the insert people need to make - guess I'll have to prettier ignore this block